sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -35,6 +35,7 @@
 #define SOF_RT5682_SSP_AMP(quirk)	\
 	(((quirk) << SOF_RT5682_SSP_AMP_SHIFT) & SOF_RT5682_SSP_AMP_MASK)
 #define SOF_RT5682_MCLK_BYTCHT_EN		BIT(9)
+#define SOF_RT5682_NUM_HDMI_DAI_FOUR		BIT(10)
```

Comment:
@sathya-nujella I have a different suggestion to add the HDMI quirk. Why not making 2 bits or maybe even larger and set it all the platforms in the quirk table? This is a bit more work but it will make the logic clearer and make it configurable in the future too.

Reply:
@ranj063 , I have updated now with HDMI quirk, similar to the other ones.
Can you please review now?